src/lang/std/engineConnection.ts

```diff
@@ -1102,7 +1106,7 @@ interface PendingMessage {
 }
 export class EngineCommandManager extends EventTarget {
   /**
-   * The artifactMap is a client-side representation of the commands that have been sent
+   * The artifactGraph is a client-side representation of the commands that have been sent
```

Comment:
Same goes for this comment and maybe a couple others around here; they need more substantive edits to be true to the newly-divided responsibilities between ArtifactGraph and ResponseMap